Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -340,6 +359,27 @@ namespace AzToolsFramework::Prefab
         return m_rootAliasFocusPathLength;
     }
 
+    void PrefabFocusHandler::SetPrefabEditScope([[maybe_unused]] AzFramework::EntityContextId entityContextId, PrefabEditScope mode)
+    {
+        m_prefabEditScope = mode;
+        SwitchToEditScope();
+    }
+
+    int PrefabFocusHandler::GetOpenInstanceMode()
+    {
+        return m_openInstanceMode;
+    }
+
+    bool PrefabFocusHandler::GetAllowContextMenuInstanceExpanding()
+    {
+        return m_allowContextMenuInstanceExpanding;
+    }
+
+    bool PrefabFocusHandler::GetContainerStepByStepSelection()
+    {
+        return m_containerStepByStepSelection;
+    }
+
```

Comment:
These were used to toggle individual features on/off during prototyping but will no longer be necessary, so they can be removed.